### DISCUSSION TITLE
[Feature Request] Deno documentation

### DISCUSSION THREAD

**Discussion by edfloreshz** [heroui-inc/heroui#4699] [Feature requests]
### Is your feature request related to a problem? Please describe.

There's currently no documentation for Deno.

### Describe the solution you'd like

Would be great to have documentation explaining how to use NextUI with Deno.

### Describe alternatives you've considered

No alternatives.

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1757/[feature-request]-deno-support">ENG-1757 [Feature Request] Deno support</a></p>
